projects/gnoland/gno.land/r/stackdump000/grc000/litepaper.md

```diff
@@ -0,0 +1,74 @@
+# Append-Only Petri-Nets for GRC Standards
+
+**Status:** Draft • **Version:** v0 • **Target chain:** gno.land
```

Comment:
This is a summary of the "append only" model updates made possible w/ metamodel composition.